app/views/admin/collections/show.html.erb

```diff
@@ -99,10 +124,112 @@ Unless required by applicable law or agreed to in writing, software distributed
 
 <%= render "form", modal_title: "Edit Collection Infomation" %>
 
+<% content_for :page_styles do %>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/cropperjs/1.5.5/cropper.min.css" integrity="sha256-mS+0WtWHI0Mw4UuIK5xU+b+pB/vfWZiEHV4sJWx7az8=" crossorigin="anonymous" />
+<% end %>
+
 <% content_for :page_scripts do %>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/cropperjs/1.5.5/cropper.min.js" integrity="sha256-vshROrHusjRqmlrpFKS7oAAd4YV7Stzn6DlDmtIBG/4=" crossorigin="anonymous"></script>
```

Comment:
Is it better to use the CDN here or manage this via yarn?

Reply:
I'd vote for `yarn` to keep the dependency management consistent when looking for stuff.

Reply:
OK. How would that be done?

Reply:
Yea, guess depends how Avalon is wired up.  In the project root in a terminal window you could try `yard add cropperjs`.  Then start up the docker container?

Here's the package:
https://www.npmjs.com/package/cropperjs

Reply:
I've added cropperjs to the yarn package, but I don't know how to import it when I need in the app. What is the path that it ends up at?

Reply:
@bkeese Did you get this figured out?

Reply:
Yeah. Thanks.